clusterloader2/pkg/prometheus/manifests/default/master-kubelet-endpoints.yaml

```diff
@@ -0,0 +1,14 @@
+# Endpoints object for the kubelet running on master node.
+apiVersion: v1
+kind: Endpoints
+metadata:
+  namespace: kube-system
+  name: master-kubelet
+  labels:
+    k8s-app: master-kubelet
```

Comment:
What interesting metrics kubelet exposes?

Reply:
To be honest I'm not sure. I believe that majority of the useful metrics come from cadvisor.
Nevertheless it doesn't cost much to scrape kubelet and it always may turn out to be useful in future.
When it comes to metrics I adhere the principle - better more than less :) 

LMK WDYT

Reply:
So the answer is "just in case we will need them" - I'm fine with that, just wanted to understand it.